zookeeper-server/src/main/java/org/apache/zookeeper/server/NIOServerCnxnFactory.java

```diff
@@ -962,14 +979,14 @@ public void dumpConnections(PrintWriter pwriter) {
     @Override
     public void resetAllConnectionStats() {
         // No need to synchronize since cnxns is backed by a ConcurrentHashMap
-        for(ServerCnxn c : cnxns){
+        for (ServerCnxn c : cnxns) {
             c.resetStats();
         }
     }
 
     @Override
     public Iterable<Map<String, Object>> getAllConnectionInfo(boolean brief) {
-        HashSet<Map<String,Object>> info = new HashSet<Map<String,Object>>();
+        HashSet<Map<String, Object>> info = new HashSet<Map<String, Object>>();
```

Comment:
Does this need to be `HashSet` or can it be `Set`?

Reply:
@enixon totally agreed that we should use the interface here

Personally I would prefer not to change this and the other points just because of checkstyle.

Let's see what the others in the community think

Reply:
I'd also prefer limiting the scope of this change to be mechanical changes that's only appertain to style and formatting.

@TisonKun since you are on this, do you mind to create a couple of JIRAs to follow up the issues around log formatting, interface declaration and other nits that @enixon pointed out?

Reply:
That's a fair preference. I'll stop pointing out these kind of nits in reviewing this diff and stick to validating correctness.